packages/react-vapor/src/components/modalWizard/tests/ModalWizard.spec.tsx

```diff
@@ -0,0 +1,132 @@
+import {screen, waitForElementToBeRemoved} from '@testing-library/dom';
+import userEvent, {specialChars} from '@testing-library/user-event';
+import * as React from 'react';
+import {renderModal} from 'react-vapor-test-utils';
+
+import {ModalWizard} from '../ModalWizard';
+
+describe('ModalWizard', () => {
```

Comment:
One use-case that I don't see here is the ability to have dynamic children.

For instance, if I check a checkbox in step 1, then it adds an additional step to the wizard.

I'd really like to know if your thing supports this, because we need it for the Commerce one :smile:

Reply:
I'll add an example